enhancements/update/change-management-and-maintenance-schedules.md

```diff
@@ -0,0 +1,1220 @@
+policies.
+
+It is thus occasionally necessary to distinguish a resource's **configured** vs **effective** change management
+state. There are two states: "paused" and "unpaused" (a.k.a. permissive; meaning that material changes be initiated). 
+For a control-plane resource, the configured and effective enforcement states are always the same. For worker-node
+resources, the configured strategy may be disabled, but the effective enforcement state can be "paused" due to 
+an active strategy in the control-plane resource being in the "paused" state.
+
+| control-plane state     | worker-node state         | worker-node effective state | results                                                                                                                                                                                            |
+|-------------------------|---------------------------|-----------------------------|----------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| unpaused                | unpaused                  | unpaused                    | Traditional, fully self-managed change rollouts. Material changes can be initiated immediately upon configuration change. |
+| paused (any strategy)   | **unpaused**              | **paused**                  | Changes to both the control-plane and worker-nodes are constrained by the control-plane strategy. |
+| unpaused                | paused (any strategy)     | paused                      | Material changes can be initiated immediately on the control-plane. Material changes on worker-nodes are subject to the worker-node policy. |
+| paused (any strategy)   | paused (any strategy)     | paused                      | Material changes to the control-plane are subject to change control strategy for the control-plane. Material changes to the worker-nodes are subject to **both** the control-plane and worker-node strategies - if either precludes material change initiation, changes are left pending. |
+
+#### Maintenance Schedule Strategy
+The maintenance schedule strategy is supported by all resources which support change management. The strategy
+is configured by specifying an RRULE identifying permissive datetimes during which material changes can be
+initiated. The cluster lifecycle administrator can also exclude specific date ranges, during which 
+material changes will be paused.
+
+#### Disabled Strategy
+This strategy indicates that no change management strategy is being enforced by the resource. It always implies that
+the enforcement state at the resource level is unpaused / permissive. This does not always
+mean that material changes are permitted due to change management hierarchies. For example, a MachineConfigPool
+with `strategy: Disabled` would still be subject to a `strategy: MaintenanceStrategy` in the ClusterVersion resource.

```

Comment:
Hoping this will clear with the current layout of change management `status` in the MCP:
```
status:
  changeManagement:
    # Always show control-plane level strategy. Disabled if disabledUntil is "true".
    clusterStrategy: <Disabled|MaintenanceSchedule>
    # If this a worker-node related resource (e.g. MCP), show local strategy. Disabled if disabledUntil is "true".
    workerNodeStrategy: <Disabled|MaintenanceSchedule|Manual|Assisted>
    # Show effective state.
    effectiveState: <Changes Paused|Changes Permitted>
    description: "Human readable message explaining how strategies & configuration are resulting in the effective state."
...
```

So in this case MaintenanceSchedule/DIsabled scenario, the user would see the cluster strategy, their MCP strategy, what is actually happening for the specific MCP, and a human readable explanation.